matrikkel-api/src/testFixtures/kotlin/no/kartverket/matrikkel/bygning/matrikkelapi/builders/DatoBuilders.kt

```diff
@@ -0,0 +1,12 @@
+package no.kartverket.matrikkel.bygning.matrikkelapi.builders
+
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.Timestamp
+import javax.xml.datatype.DatatypeFactory
+
+private val datatypeFactory by lazy { DatatypeFactory.newDefaultInstance() }
+
+fun timestampUtc(year: Int, month: Int, day: Int, hour: Int = 0, minute: Int = 0, second: Int = 0): Timestamp {
```

Comment:
Detalj, men gjøre om denne til samme mønster som de andre?

```suggestion
fun timestampUtc(year: Int, month: Int, day: Int, hour: Int = 0, minute: Int = 0, second: Int = 0): Timestamp = Timestamp()
    .apply {
        timestamp = datatypeFactory.newXMLGregorianCalendar(year, month, day, hour, minute, second, 0, 0)
    }
```

Reply:
Ja. Grunnen til at det ikke ble sånn fra starten av var at jeg opprettet datatypeFactory i funksjonen. Jeg endret på det fort, men overså muligheten til å da forenkle.